central/scannerdefinitions/handler/handler_test.go

```diff
@@ -306,50 +394,89 @@ func (s *handlerTestSuite) TestServeHTTP_v4_Offline_Get() {
 	err = s.mockHandleZipContents(filePath)
 	s.Require().NoError(err)
 
-	req = s.getRequestWithVersionedFile(t, "4.3.0")
-	w = httptest.NewRecorder()
+	// This will fail because 4.5.0 uses the multi-bundle ZIP format.
+	req = s.getRequestVersion("4.5.0")
+	h.ServeHTTP(w, req)
+	s.Equal(http.StatusNotFound, w.Code)
+
+	// Set the header properly.
+	req.Header.Set("X-Scanner-V4-Accept", "application/vnd.stackrox.scanner-v4.multi-bundle+zip")
```

Comment:
```suggestion
	w = httptest.NewRecorder()
	req.Header.Set("X-Scanner-V4-Accept", "application/vnd.stackrox.scanner-v4.multi-bundle+zip")
```
Adding this should fix one of the unit test failures, looks like the `404` from the previous call to `h.ServeHTTP()` is being carried forward  in the recorder.

Reply:
I did this to myself: https://github.com/stackrox/stackrox/pull/12210 :(